README.md

```diff
@@ -276,6 +276,9 @@ kubescape submit results path/to/results.json
 
 Scan the YAML files while writing them using the [vs code extension](https://github.com/armosec/vscode-kubescape/blob/master/README.md) 
 
+## Lens Extension
```

Comment:
`View Kubescape scan results directly in [Lens IDE](https://k8slens.dev/) using` ~our~ -> *kubescape* `[Lens extension](https://github.com/armosec/lens-kubescape/blob/master/README.md)`